apps/studio/prisma/schema.prisma

```diff
@@ -1,4 +1,4 @@
-// This is your Prisma schema file,
+// This is your Prisma schema file,/
```

Comment:
nit: extra slash

Reply:
idk how that got there but thanks for the catch